### PR TITLE
Documentation for deleting files

### LINKED ISSUE #726: 📕Delete Parse.file
__References:__
parse-community/Parse-SDK-JS#1067

__Proposed solution:__
should go here -> https://docs.parseplatform.org/js/guide/#retrieving-file-contents

Also the header needs a space...
IMG_0842

### PR DESCRIPTION
**Pull request by dblythy**
Closes #726

### CODE REVIEW

**Review by davimacedo — APPROVED** [review]
It looks good to me. @TomWFox any comment?

File: _includes/js/files.md [hunk 1]
```diff
@@ -111,9 +111,22 @@ Parse.Cloud.httpRequest({ url: profilePhoto.url() }).then(function(response) {
 });
 ```
 
+
 ## Deleting Files
 
-You can delete files that are referenced by objects using the [REST API]({{ site.baseUrl }}/rest/guide/#deleting-files). You will need to provide the master key in order to be allowed to delete a file.
+Starting with Parse Server 4.2.0, you can delete files using cloud code.
```
Comment (mtrezza) [inline _includes/js/files.md]: This sounds like deleting files was only possible since Parse Server 4.2. I suggest rewording that it is clear that prior to 4.2. the REST API can be used.

**Review by mtrezza — CHANGES_REQUESTED** [review]

File: _includes/js/files.md [hunk 2]
```diff
@@ -111,12 +111,23 @@ Parse.Cloud.httpRequest({ url: profilePhoto.url() }).then(function(response) {
 });
 ```
 
+
```
Comment (mtrezza) [inline _includes/js/files.md]: Empty line should be removed.

**Review by mtrezza — APPROVED** [review]

**Comment by mtrezza** [comment]
@TomWFox I just merged this and in the moment I thought you may wanna look over it, apologies if that was off protocol.

**Comment by mtrezza** [comment]
My meticulousness kicked in and I opened https://github.com/parse-community/docs/pull/791 to do some minor corrections:
- Clarified wording; it could be understood that for Parse Server <4.2.0 it is not possible to delete a file using Cloud Code, which is incorrect.
- Simplified example code; it is not necessary to use a `beforeDelete` hook the example; given the `beforeDelete` name, one could assume that it is somehow related to the basic functionality of deleting a file, which it is not.

**Comment by dblythy** [comment]
I'm glad you accidentally merged this because you've done a better job than I did haha. Thanks for your hard work for the community!

**Comment by mtrezza** [comment]
I only realized these subtleties after merging, otherwise I'd have suggested the changes to you. Thanks for opening this PR!

**Comment by TomWFox** [comment]
@mtrezza Ideally I like to look at all the docs PRs before merging but its not a big deal :)

**Comment by mtrezza** [comment]
I though so, next time ;)
